src/Microsoft.ApplicationInsights.AspNetCore/Implementation/TelemetryConfigurationOptionsSetup.cs

```diff
@@ -95,25 +95,25 @@ public void Configure(TelemetryConfiguration configuration)
 
         private void AddTelemetryChannelAndProcessorsForFullFramework(TelemetryConfiguration configuration)
         {
+            // Enabling Quick Pulse Metric Stream
+            if (this.applicationInsightsServiceOptions.EnableQuickPulseMetricStream)
```

Comment:
so it is a change in behavior. For full framework fro some reasons we didn't enable QP for non server channel. Perhaps make the same change for sampling then

Reply:
Done